packages/app-store/routing-forms/components/FormInputFields.tsx

```diff
@@ -73,7 +73,7 @@ export default function FormInputFields(props: FormInputFieldsProps) {
                     ...response,
                     [field.id]: {
                       label: field.label,
-                      identifier: field?.identifier,
+                      identifier: field?.identifier || null,
```

Comment:
If we are relying on identifier to be always set then I think we should use https://github.com/calcom/cal.com/blob/add-teams-routing-forms-response-record-endpoint-back/packages/app-store/routing-forms/lib/getFieldIdentifier.ts#L3